docker-compose.production.yaml

```diff
@@ -4,8 +4,10 @@ services:
 
   app:
     image: usedgoodreads/app:production
-    ports:
-      - "80:5000"
 
   worker:
     image: usedgoodreads/app:production
+
+  nginx:
+    volumes:
+      - /etc/letsencrypt/live/usedgoodreads.com/:/certs
```

Comment:
We should have some docs wrt. how to set this up on the remote host in the readme.

Reply:
Added to the README in f051b0811cfc7296f8f086aaa90289d19564b457